source/CardGames.py

```diff
@@ -32,6 +32,10 @@ def __eq__(self, other):
     return self.suit == other.suit and \
       self.value == other.value
 
+  def setValue(self):
+    if self.value > 10:
+      self.value = 10
+    return self.value
 
```

Comment:
Hi Joanna! Sorry I know I'm not assigned to comment on yours, but you are the next person in line for me to comment on.ut your code looks good! I'm nit sure if there is a simpler way to set the value =10